BuildAddOn.py

```diff
@@ -16,9 +16,10 @@ def ParseArguments ():
     parser = argparse.ArgumentParser ()
     parser.add_argument ('-c', '--configFile', dest = 'configFile', required = True, help = 'JSON Configuration file')
     parser.add_argument ('-v', '--acVersion', dest = 'acVersion', nargs = '+', type = str, required = False, help = 'Archicad version number list. Ex: 26 27')
-    parser.add_argument ('-l', '--allLocalizedVersions', dest = 'allLocalizedVersions', required = False, action='store_true', help = 'Create localized release builds for all configured languages.' )
+    parser.add_argument ('-b', '--buildConfig', dest = 'buildConfig', nargs = '+', type = str, required = False, help = 'Build configuration list. Ex: debug RELEASE RelWithDebInfo')
+    parser.add_argument ('-l', '--allLocalizedVersions', dest = 'allLocalizedVersions', required = False, action='store_true', help = 'Create localized release builds for all configured languages.')

```

Comment:
We can just go with the exact string, no need to handle casing.

Reply:
It's better for the user experience if we handle casing. Also, the resulting build folders will always have the configuration name in upper camel case, so that has to be handled anyway.

Reply:
I'm not sure it's better. From the code perspective it causes more complex logic, you need to think about this and handle everywhere. From the user's perspective they will provide a parameter, and the resulting folder will be different, I won't call it good user experience. I think it's more clean and easy to use if you have to provide the exact word with the exact casing.

Reply:
Removed case handling